iosApp/iosApp/LaunchView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LaunchView.swift
+//  iosApp
+//
+//  Created by Nasir Ahmed Momin on 20/05/22.
+//  Copyright © 2022 Mutual Mobile. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct LaunchView: View {
+    
+    @State private var selection = 0
+    
+    private var textList = [
+        "Simple, powerful time tracking, reporting and invoicing",
+        "Track time easily, wherever you are",
+        "Enter expanses on go",
+        "Report on time for powerful insights"
+    ]
+    
+    var body: some View {
+        
+        GeometryReader { proxy in
```

Comment:
GeometryReader is correct but we have faced problems with GeometryReader in past. Maybe those are not present anymore in latest versions. When whole screen height and width is required, see if we can use `UIScreen.main.bounds`

Reply:
@chitvanMM Thank for review.

There are 2 reason to use `GeometryReader` here, 
1. To full background colour, without it, code would have require some kind of hack with `ZStack`
2. To get width of screen which I replaced with you suggestion of `UIScreen.main.bounds` Thanks Its perfectly suited in this situations


All these new changes is in a new branch named `feature/iOS-Login` & will be available in next MR.